docs/code_push/release.mdx

```diff
@@ -135,7 +135,7 @@ shorebird release android --force
 or
 
 ```
-shorebird release ios-alpha --force
+shorebird release ios --force
```

Comment:
I can't comment on the unchanged lines above, but we really need to remove `--force` guidance from our docs 😬

Reply:
Yeah agreed just didn't want to blow up the scope of this PR

Reply:
100%, just making note